### DISCUSSION TITLE
use getKEGGModelForOrganism with KEGG release 100

### DISCUSSION THREAD

**Discussion by FengxuSysbio** [SysBioChalmers/RAVEN#398] [Help me!] [answered]
When I start to generate the model by using the function using the version of RAVEN 2.5.3, the error was "The file 'gene.pep' cannot be located at prok90_kegg100 and should be downloaded from the KEGG FTP". How could I solve this question? Thanks!

**Comment by edkerk** [comment, marked as answer]
You should be using [version 2.6.0](https://github.com/SysBioChalmers/RAVEN/releases/tag/v2.6.0) if you want to use the HMMs trained by KEGG version 100. What is the exact command that you run?

**Reply by FengxuSysbio** [reply]
Thanks for your suggestion. The command is `PpaKEGGHomology2=getKEGGModelForOrganism('PpaKEGGHMMs','GCF_000027005.1_ASM2700v1_protein.faa','prok90_kegg100','',1,0,0)`. And my matlab version is 2021b. I am going to try the new version of RAVEN.

**Reply by edkerk** [reply]
There shouldn't be anything wrong with your command, so most likely it's solved with using RAVEN 2.6.0. Do please report whether your problem was resolved :)

**Reply by FengxuSysbio** [reply]
Thanks for your suggestions. The problem was solved.

**Reply by edkerk** [reply]
@FengxuSysbio Can you please open a new discussion for a new problem? Thank you!
